tests/lib/rules/jsx-boolean-value.js

```diff
@@ -110,11 +122,11 @@ ruleTester.run('jsx-boolean-value', rule, {
       errors: [
         {
           messageId: 'setBoolean',
-          data: { exceptionsMessage: ' for the following props: `foo`, `bar`' },
+          data: { exceptionsMessage: ' for the following props: `foo`' },
         },
         {
           messageId: 'setBoolean',
-          data: { exceptionsMessage: ' for the following props: `foo`, `bar`' },
+          data: { exceptionsMessage: ' for the following props: `bar`' },
```

Comment:
this wording suggests it can be a list, which means there's no reason to split it into two messages.

If it's kept as separate messages (which i understand would be a better DX, especially for multiline props), then it should say ``` for the `foo` prop ``` or something.

Reply:
you're right. It seems like I missed something in the process of separating the error message.

i'll change the message to `for the `foo` prop`